src/web/js/output-ui.js

```diff
@@ -926,6 +969,9 @@
             }, "optional: help(contents)");
           },
           "text": function(txt) {
+            if (cpoDialect === "patch") {
```

Comment:
I think I would prefer to see a `$cpoPyret` and `$cpoPatch` pair of renderers, rather than a single renderer that's polyglot across the two.  You can have `$cpoPatch` extend from `$cpoPyret` and override just the parts that change...

Reply:
(The intent of making a dictionary of these various renderers was that it should be easy to add new ones for new contexts, not shoehorn new features into old renderers.)